app/models/olympian.rb

```diff
@@ -26,4 +26,18 @@ def self.oldest
     Olympian.order(age: :desc).take
   end
 
+  def self.average_weight(sex = nil)
+    if sex == "males"
+      Olympian.where(sex: "M").average(:weight)
+    elsif sex == "females"
+      Olympian.where(sex: "F").average(:weight)
+    else
+      Olympian.average(:weight)
+    end
+  end
+
+  def self.average_age
+    Olympian.average(:age)
```

Comment:
This would be a good method to circle back to and give it the ability to return average age by sex (like how average_weight can take an argument)